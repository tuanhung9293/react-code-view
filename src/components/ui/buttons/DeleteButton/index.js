import React from 'react'
import IconButton from '../IconButton'
import trash from 'assets/trash.svg'

function DeleteButton(props) {
  return <IconButton {...props}><img src={trash} alt="Add to favorite" /></IconButton>
}

export default DeleteButton