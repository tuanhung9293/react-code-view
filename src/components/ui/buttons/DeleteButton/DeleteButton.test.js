import React from 'react'
import {shallow} from 'enzyme'
import DeleteButton from './'

it('renders as a DeleteButton', () => {
  expect(shallow(
    <DeleteButton />
  )).toMatchSnapshot()
})

it('should DeleteButton pass props', () => {
  expect(shallow(
    <DeleteButton dataTest="dataTest" />
  )).toMatchSnapshot()
})
