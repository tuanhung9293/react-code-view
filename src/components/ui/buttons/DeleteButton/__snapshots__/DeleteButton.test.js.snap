// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as a DeleteButton 1`] = `
<IconButton>
  <img
    alt="Add to favorite"
    src="test-file-stub"
  />
</IconButton>
`;

exports[`should DeleteButton pass props 1`] = `
<IconButton
  dataTest="dataTest"
>
  <img
    alt="Add to favorite"
    src="test-file-stub"
  />
</IconButton>
`;
